kart/completion.py

```diff
@@ -100,6 +101,8 @@ def install_bash(*, prog_name: str, complete_var: str, shell: str) -> Path:
     # But installing in the locations from the docs doesn't seem to have effect
     completion_path = Path.home() / f".bash_completions/{prog_name}.sh"
     bashrc_path = Path.home() / ".bashrc"
+    if platform.system() == "Darwin":
```

Comment:
Maybe we should be checking what exists - if user has one and not the other, probably it's good to append it to the one they have?